core/tests/client.rs

```diff
@@ -22,6 +22,14 @@ fn test_rpc_client() {
         solana_clap_utils::version!()
     );
 
+    match client.get_account(&bob_pubkey) {
```

Comment:
`assert!(client.get_account(&bob_pubkey).is_err());` is all we need here, I don't think it matters to assert the exact error message.

Reply:
LOL I had that originally and changed it to match the server error 😂